mesonbuild/build.py

```diff
@@ -1195,8 +1239,9 @@ def process_kwargs(self, kwargs):
             if not os.path.isfile(trial):
                 raise InvalidArguments(f'Tried to add non-existing resource {r}.')
         self.resources = resources
-        if kwargs.get('name_prefix') is not None:
-            name_prefix = kwargs['name_prefix']
+        # TODO: the wallrus operator would reduce some pain here
```

Comment:
Just one `l` in walrus I think... Not sure this comment is really useful...

Reply:
Dropped the comment in the latest version.